step.yml

```diff
@@ -105,9 +105,9 @@ inputs:
       category: "Debug"
       title: "Test timeout"
       summary: |
-        The max time this test execution can run before it is cancelled. It does not include any time necessary to prepare and clean up the target device. The maximum possible testing time is 3600 seconds.
+        The max time this test execution can run before it is cancelled. It does not include any time necessary to prepare and clean up the target device. The maximum possible testing time is 1800 seconds.
```

Comment:
Summary and Description are the same, maybe we should use the first sentence as the summary and the following as the description.